Source/WebKit/UIProcess/API/C/playstation/WKView.cpp

```diff
@@ -174,9 +167,7 @@ void WKViewSetViewClient(WKViewRef view, const WKViewClientBase* client)
             if (!m_client.enterFullScreen)
                 return completionHandler(false);
             m_client.enterFullScreen(WebKit::toAPI(&view), m_client.base.clientInfo);
-
-            // FIXME: Replace this and WKViewWillEnterFullScreen with a listener object.
-            completionHandler(false);
+            completionHandler(true);
```

Comment:
Was the old code incorrect, or has the meaning of the completion handler parameter changed? In either case, it would be a good idea to mention this change in the change log for future code spelunkers.